src/App.tsx

```diff
@@ -26,11 +29,28 @@ function BridgeApp () {
     return false;
   }
 
+  useEffect(() => {
+    const polkadotExtension = async () => {
+      const extensions = await web3Enable('asset-transfer-ui');
+
+      if (extensions.length > 0) {
+        enablePolkaEth(true);
+      }
+
+    }
+
+    polkadotExtension();
+  });
+
   if (!ethEnabled()) {
     alert("Please install MetaMask to use this application!");
     return(<div><p>Please install MetaMask to use this application!</p></div>)
   }
 
+  if(!polkaEthEnabled) {
+    return(<div><p>Please install the Polkadotjs Browser Extension!</p></div>)
```

Comment:
we should probably have a button here to try connect to polkadotjs again if first time fails or something

Reply:
Good idea.